src/app.rs

```diff
@@ -71,11 +72,19 @@ impl App {
             _ => 1,
         };
 
+        let db_connection_timeout = match (env::var("DB_TIMEOUT"), config.env) {
+            (Ok(num), _) => num.parse().expect("couldn't parse DB_TIMEOUT"),
```

Comment:
in the long run we might want to make this error message a little better but it's fine for now